docs/moduleIn3Utils.rst

```diff
@@ -0,0 +1,78 @@
+in3Utils: Module Project Utilities - generateTopo.py
+====================================================
+
+Generate DEM files for idealised/generic topographies that can be used as required DEM input data for snow avalanche simulations.
+
+The following topography types can be chosen:
+
+* flat plane
+* inclined plane of constant slope
+* hockey stick - using a parabola to describe the sloping plane that then transitions into a flat foreland
+* hockey stick smooth - using an linearly sloping plane with smoothed transition to the flat foreland
+* bowl-shaped topography
+* helix-shaped topography
```

Comment:
Add the short codes used in the ini file

Reply:
Added short codes